pkg/util/versionchecker/BUILD.bazel

```diff
@@ -0,0 +1,104 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+# Clone empty version of cert-manager repo and list all tags
+genrule(
+    name = "git_tags",
+    outs = [":git_tags.txt"],
+    cmd = "git ls-remote -t --refs https://github.com/jetstack/cert-manager.git | awk '{print $$2;}' | sed 's/refs\\/tags\\///' | sed -n '/v1.0.0-alpha.0/,$$p' > $@",
+)
+
+genrule(
+    name = "test_manifests",
+    srcs = [
+        ":git_tags.txt",
+        "//deploy/manifests:cert-manager.yaml",
+        "//:version",
+    ],
+    outs = ["test_manifests.tar"],
+    cmd = """
+CURRENT_VERSION=$$(cat $(location //:version))
+cp $(location //deploy/manifests:cert-manager.yaml) "$$CURRENT_VERSION.yaml"
+for tag in $$(cat $(location :git_tags.txt))
+do
+    # The "v1.2.0-alpha.1" manifest contains duplicate crds, skip for tests
+    if [[ $$tag == "v1.2.0-alpha.1" ]]; then
+        continue
+    fi
```

Comment:
The manifest https://github.com/jetstack/cert-manager/releases/download/v1.2.0-alpha.1/cert-manager.yaml contains duplicate definitions for the CRD resources. This causes the test to fail, because it cannot add resources that were added already. Since it is just an error in the manifests, I skipped the manifest instead of figuring out how to handle this edge-case in the tests.

Reply:
I think I misread the code. I see now that this does skip downloading that version. Fine.
I was probably thinking that `v1.2.0-alpha.1` could be grepped out of the `ls-remote` output above.
Ignore my comment.